frontend/src/app/organization/organization-contact/organization-contact.component.ts

```diff
@@ -0,0 +1,30 @@
+import {Component, EventEmitter, Input, OnInit, Output, ViewChild} from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { OrganizationContactPayload } from '../organization.model';
+
+@Component({
+  selector: 'volontulo-organization-contact',
+  templateUrl: './organization-contact.component.html',
+  styleUrls: ['./organization-contact.component.css']
+})
+export class OrganizationContactComponent implements OnInit {
+  @ViewChild('contactForm') contactForm: NgForm;
+  @Output() contact = new EventEmitter<OrganizationContactPayload>();
+  @Input() contactStatus: Number;
+
+  constructor() { }
+
+  ngOnInit() {
```

Comment:
can we stick with one convention here? `constuctor` looks like:
```
constructor() { }
```
and `ngOnInit` looks like:
```
ngOnInit () {
}
```

Reply:
empty constructor is not needed.
empty ngOnInit (and the `implements OnInit`) are not neede even more.